src/sentry/integrations/github/issues.py

```diff
@@ -205,6 +205,24 @@ def get_link_issue_config(self, group: Group, **kwargs: Any) -> list[dict[str, A
         autocomplete_url = reverse(
             "sentry-integration-github-search", args=[org.slug, self.model.id]
         )
+        if group.issue_category == GroupCategory.FEEDBACK:
+            comment = (
+                "Sentry feedback: [{issue_id}]({url})".format(
+                    url=absolute_uri(
+                        group.get_absolute_url(params={"referrer": "github_integration"})
+                    ),
+                    issue_id=group.qualified_short_id,
+                ),
+            )
+        else:
+            comment = (
+                "Sentry issue: [{issue_id}]({url})".format(
+                    url=absolute_uri(
+                        group.get_absolute_url(params={"referrer": "github_integration"})
+                    ),
+                    issue_id=group.qualified_short_id,
+                ),
+            )
```

Comment:
it's probably worth doing [extract method](https://refactoring.guru/extract-method) refactor here, as these lines are nearly same and would be nice to group logic together.

if we end up doing more formatting changes based on grouptype, it could be worth introducing some kind of class based abstraction